package.json

```diff
@@ -1,19 +1,20 @@
 {
-  "name": "vue-svg-loader",
+  "name": "vue-svg-loader-2",
   "description": "Use SVG files as Vue Components",
-  "version": "0.17.0-beta.2",
+  "version": "0.18.0-beta.1",
   "keywords": [
+    "vue-svg-loader-2",
     "vue-svg-loader",
     "svg",
     "svgo",
     "loader",
     "webpack",
     "vue"
   ],
-  "bugs": "https://github.com/visualfanatic/vue-svg-loader/issues",
-  "repository": "visualfanatic/vue-svg-loader",
+  "bugs": "https://github.com/hfotechnology/vue-svg-loader-2/issues",
+  "repository": "hfotechnology/vue-svg-loader-2",
   "license": "MIT",
-  "author": "Damian Stasik <npm@coded.pl>",
+  "author": "Gamma Communications",
```

Comment:
Please use the full company name "Gamma Communications GmbH" here as required by law. Also, you can use the e-mail address webdevelopment@gammacommunications.de here.